Engine/DataFeeds/BacktestingOptionChainProvider.cs

```diff
@@ -51,13 +56,19 @@ public IEnumerable<Symbol> GetOptionContractList(Symbol underlyingSymbol, DateTi
                 throw new NotSupportedException($"BacktestingOptionChainProvider.GetOptionContractList(): SecurityType.Equity, SecurityType.Future, or SecurityType.Index is expected but was {underlyingSymbol.SecurityType}");
             }
 
+            // Resolve any mapping before requesting option contract list
+            var mapFileResolver = _mapFileProvider.Get(underlyingSymbol.ID.Market);
+            var mapFile = mapFileResolver.ResolveMapFile(underlyingSymbol.Value, date);
```

Comment:
Think should send in `.ID.Symbol` here, can see other users of `.ResolveMapFile()`

Reply:
Adding on my review to this particular section of code, unrelated to MM's comment:

Are we sure we want to map futures/index options? 

The OptionChainProvider implementations were adjusted to generically support any kind of option chain being requested from it, regardless of SecurityType. 

Maybe we could have (or already have) some sort of extension method that checks if a Symbol or SecurityType requires mapping? 

Not a change that has to be added, but I do want to make sure we don't map Symbols for assets that don't need to be.